packages/core/src/routes/custom-phrase.ts

```diff
@@ -70,6 +72,17 @@ export default function customPhraseRoutes<T extends AuthedRouter>(router: T) {
         params: { languageKey },
       } = ctx.guard;
 
+      const {
+        languageInfo: { fallbackLanguage },
+      } = await findDefaultSignInExperience();
+
+      if (fallbackLanguage === languageKey) {
+        throw new RequestError({
+          code: 'localization.cannot_delete_default_language',
+          languageKey,
+        });
```

Comment:
@xiaoyijun   should we also block the use from clicking on the delete button at the AC client side as well

Reply:
Yes. Tracked: https://linear.app/silverhand/issue/LOG-4208/not-allow-to-delete-default-language-in-manage-language